src/VisualStudio/Xaml/Impl/project.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+  "dependencies": {
+    "NuGet.VisualStudio": {
```

Comment:
I don't see where you need this, and given that, you can probably remove the embedinteroptypes goo from the project file.